src/services/routePathLinkService.ts

```diff
@@ -5,41 +5,90 @@ import { NeighborToAddType } from '~/stores/routePathStore';
 import ErrorStore from '~/stores/errorStore';
 import TransitType from '~/enums/transitType';
 import RoutePathLinkFactory from '~/factories/routePathLinkFactory';
+import INeighborLink from '~/models/INeighborLink';
+import IGraphqlList from '~/models/externals/graphqlModelHelpers/IGraphqlList';
+import IExternalNode from '~/models/externals/IExternalNode';
 import IExternalLink from '~/models/externals/IExternalLink';
 import GraphqlQueries from './graphqlQueries';
 
+interface ISimplifiedRoutePath {
+    reitunnus: string;
+}
+
+interface IExtendedExternalNode extends IExternalNode {
+    usageDuringDate?: IGraphqlList<ISimplifiedRoutePath>;
+}
+
+interface IExtendedExternalLink extends IExternalLink {
+    solmuByLnkalkusolmu: IExtendedExternalNode;
+    solmuByLnkloppusolmu: IExtendedExternalNode;
+}
+
+interface IFetchNeighborLinksResponse {
+    neighborToAddType: NeighborToAddType;
+    neighborLinks: INeighborLink[];
+}
+
+const getNeighborLinks = (
```

Comment:
Move most of the content in this service to a new service: RoutePathNeighborLinkService?